ui/redux/rootReducer.js

```diff
@@ -310,6 +311,26 @@ export const updateLocusList = values => (dispatch) => {
     }).post(values)
 }
 
+export const loadClinGeneAlleleId = hgvsc => (dispatch, getState) => {
+  if (!hgvsc) {
+    return
+  }
+  const alleleId = getState().clinGenAlleleIdByHgvsc[hgvsc]
+  if (!alleleId) {
+    dispatch({ type: REQUEST_CLINGEN_ALLELE_ID })
+    new HttpRequestHelper('http://reg.genome.network/allele',
+      (responseJson) => {
+        dispatch({
+          type: RECEIVE_DATA,
+          updatesById: { clinGenAlleleIdByHgvsc: { [hgvsc]: { alleleId: responseJson['@id'].split('/').pop() } } },
+        })
+      },
+      (e) => {
+        dispatch({ type: RECEIVE_DATA, error: e.message, updatesById: {} })
+      }).get({ hgvs: hgvsc }, 'omit')
```

Comment:
Add `omit` credentials to allow CORS going through.

Reply:
The `https://gtexportal.org` responses with `access-control-allow-origin: https://seqr.broadinstitute.org` (or access-control-allow-origin: http://localhost:3000 for my local seqr) while `http://reg.genome.network` responses with `Access-Control-Allow-Origin: *`. According to the [`fetch()` API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#sending_a_request_with_credentials_included) of the browser, Access-Control-Allow-Origin is prohibited from using a `*` for requests with credentials: 'include'.
Calling the API with credentials: 'omit' is safer because no authentication information, such as cookies, will be included in the requests.